### ISSUE TITLE
title bar and cell border slipped when use with other unicode language

### ISSUE THREAD

**Issue by singnoi** [apgapg/json_table#12]
<img width="354" alt="ae1" src="https://user-images.githubusercontent.com/59789789/74106830-82d1f280-4b9c-11ea-941e-4de90f7f036f.png">

like this 
can I fix it by change Alignment or padding of  tableCellBuilder ?

**Comment by apgapg** [comment]
Yes i think in this case you need to provide table cell builder and adjust its height/padding as per required

**Comment by mediocre-mayda** [comment]
I have this issue with Arabic characters, please instruct me in how I can solve this, meaning making all the columns the same height or getting rid of the extra padding that seems to come Arabic characters

**Comment by singnoi** [comment]
> Yes i think in this case you need to provide table cell builder and adjust its height/padding as per required

thank you, but this problem still persists
I think ( from my rookie experience), the row height of cells in same row would be equal if use the row widget
but I'm too new to fix 

thank you

**Comment by apgapg** [comment]
@singnoi  Do one thing use table cell builder and use a container of fixed height lets say 70 or 100 . In this way each row cell would be of fixed height and using other language wont affect it

**Comment by apgapg** [comment]
@singnoi 
```
tableCellBuilder: (value) {
    return Container(
      height: 100,
      padding: EdgeInsets.symmetric(horizontal: 4.0, vertical: 2.0),
      decoration: BoxDecoration(border: Border.all(width: 0.5, color: Colors.grey.withOpacity(0.5))),
      child: Text(
        value,
        textAlign: TextAlign.center,
        style: Theme.of(context).textTheme.display1.copyWith(fontSize: 14.0, color: Colors.grey[900]),
      ),
    );
  }
```
